themes/demula/demula.theme.bash

```diff
@@ -10,40 +11,18 @@
 # Screenshot: http://goo.gl/VCmX5
 # by Jesus de Mula <jesus@demula.name>
 
-# For the real Monokai colors you should add these to your .XDefaults or
```

Comment:
ShellCheck falsely identified these lines as shebangs. Since the comment seemed unimportant and general (this would apply to all themes), these lines were removed.

Reply:
That works!